integration_tests/server/routes/index.ts

```diff
@@ -1,12 +1,11 @@
 import { type RequestHandler, Router } from 'express'
 
-import asyncMiddleware from '../middleware/asyncMiddleware'
 import type { Services } from '../services'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export default function routes(service: Services): Router {
   const router = Router()
-  const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  const get = (path: string | string[], handler: RequestHandler) => router.get(path, handler)
```

Comment:
these `get`/`post` helpers now don’t do anything other than hide the true interface of the router methods.

i kinda wanna get rid of them if there’s appetite…?